_editorial_policies/03_reviewer_information.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: single
+sidebar:
+  nav: reviewer_information.md
+title: Information for Reviewers
+excerpt: Information for reviewers
+permalink: /policies/reviewer_information/
+---
+
+This provides information relevant to reviewers of LiveCoMS, as well as a copy of the form letters that we typically use when corresponding with reviewers.
+
+
+## Brief overview of the review process
+
+Before sending manuscripts for review, a lead editor of the relevant LiveCoMS section will have responded favorably to a [presubmission inquiry](https://livecomsjournal.github.io/authors/policies/#presubmission-letter) and then checked that the submitted manuscript indeed conforms to what was originally proposed.
+A LiveCoMS editor will also have checked that the manuscript appears to be reasonably well edited and ready for review.
+Given this preparation, LiveCoMS asks that reviewers respond promptly to review requests and take immediate action on reviewing manuscripts; typically we ask that reviews be returned within 10 days, though exceptions can be made in some cases.
+The review process is in general similar to typical journals, in that we seek to ensure manuscripts are of high quality and will provide significant benefits to the field.
+However, we do also have specific [review criteria](https://livecomsjournal.github.io/authors/policies/#review-criteria) and we ask reviewers to make sure they are aware of these, including [category-specific review criteria](https://livecomsjournal.github.io/authors/policies/#types-of-articles) relevant to the particular category of article.
```

Comment:
Add: "Consistent with its pedagogical mission, LiveCoMS requires a student reivew of each manuscript."

Reply:
Done.